elements/multicolumn/README.md

```diff
@@ -0,0 +1,54 @@
+Multicolumns are the main elements to create complex layouts. They can have multiple columns (we support max. 5 in our editor) and every columns can have children. The children array can contain every element except the full width container. [Read more](//fullwidth_container/README.md)
+
+The arrays in the “cols” array represent the columns. The minimum number of columns is 2 and it can be grown to 5.
+
+The columns can't have padding, margin, background and border properties, these properties need to be added to the elements inside the children array. 
+
+The width of the multicolumn is set to 100% and extends to the full width of its parent element. The width of the columns is calculated from the width of the multicolumns parent elements width.
+Keep in mind that: 
+ - the sum of the child elements width (with the padding, margin and border width values) won't exceed the parent elements width, because the last column might break into a new line
```

Comment:
Very nice, although I would emphasize that you are talking about the child elements of the columns